CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(sparse_bundle_adjustment)
 
+add_compile_options(-std=c++11)
```

Comment:
The CMake recommended way is now:

```
if(NOT CMAKE_CXX_STANDARD)
  set(CMAKE_CXX_STANDARD 11)
endif()
```

This works cross platform and allows to override it if necessary.

Reply:
Ah I thought I remembered seeing  a recommended way...
I just checked tf and went with the way it was done there.